gallery/src/components/NxTable/NxTableClickableExample.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
```

Comment:
I assume this is auto generated but should we be 2020 now?=)  Just noticed randomly, definitely not a blocker.

Reply:
That's why it's `-present`. I just copy-pasted the license header and I assume our license check would fail if I dared to change it.

Reply:
I guess I assumed that present covered us for all time but I am skeptical we'd be able to copyright new code in the past without someone legal saying that.
You're right tho, needs to pass check, so beyond scope...just stuck out in my mind :)

Reply:
I brought this up many years ago but its a limitation in the maven license plugin, so we typically use the inception of the project in the header instead of the creation of the code/file.